app/elements/app-element.html

```diff
@@ -33,6 +38,14 @@
 		Polymer({
 			is: 'app-element',
 			properties: {
+				apiConfig: {
+					type : Object,
+					notify: true,
+					value: {
+						apiKey: 'AIzaSyCnqlcAd2M-qW20SDsghrBOOLvs144c3og',
+						version: '3.exp'
```

Comment:
Nothing specific. I tested now and everything works with 3.24, so i'll change to that since it's most likely more stable than the "experimental" version